cmake/README.md

```diff
@@ -0,0 +1,6 @@
+# wolfSSL CMake
+
+This directory contains some supplementary functions for the [CMakeLists.txt](../CMakeLists.txt) in the root.
+
+See also the [README_cmake.md](../README_cmake.md) file.
```

Comment:
Fix. No longer README_cmake.md.

Reply:
I updated that to instead reference the root `INSTALL` document.